Gemfile

```diff
@@ -1,17 +1,18 @@
 source 'https://rubygems.org'
 
 gem 'berkshelf', '~> 4.3'
-gem 'chef', '~> 12.4'
+gem 'chef', '~> 12.4', '<= 12.13.37'
```

Comment:
I ran into another issue with cerner_splunk where it pulled in 12.14.60 version of chef gem which required ruby>2.2.0 which broke the travis build.